### ISSUE TITLE
Accessibility Window is CUmbersome to Use in Play

### ISSUE THREAD

**Issue by BlindGuyNW** [MegaMek/megamek#2636]
### Environment
What version of MegaMek does your issue apply to?  0.47.17 snapshot.
What operating system are you using?  Mac, though applies equally to all.
What java version are you using?  Java version 1.8.0_282, though applies to all.

### Description

The current game flow for playing with he accessibility window is quite tedious. In order to figure out what's going on, you need to move from the window to the main game log, and back, many times in a given turn.

Steps:
1. Open Megamek.
2. Start a game using the accessibility window.
3. Open the accessibility window form the view menu.
4. Try to deploy your units and play a turn.
5. Observe the window and what appears where.

Actual Results
Some stuff appears in the accessibility window, but it's not the game log which appears separately. A user has to switch windows often to read firing results, see how they did on PSRs, etc.

Expected Results:
The game log should be piped to the accessibility window in addition to a user's command responses. Thus, the window is used for more things but the process of play is a little easier.

Resolutions:
If there is a wayt to have the window listen to game log events and pipe them into the output stream that would be ideal.
IF not, some kind of game log event listener class could be created to allow this.
